pixeltable/exprs/function_call.py

```diff
@@ -276,6 +276,30 @@ def reset_agg(self) -> None:
         assert isinstance(self.fn, func.AggregateFunction)
         self.aggregator = self.fn.agg_class(**self.agg_init_args)
 
+    @property
```

Comment:
We need to get rid of properties that can result in exceptions (here and everywhere else, but we should definitely start here). The reason is that it can crash the debugger, because properties are shown automatically in the inspection window.

Reply:
I don't know if I agree with that. It means we're making decisions about our codebase design based on some peculiarity of a particular debugger. That's *really* putting the cart before the horse. If you're counting assertions as exceptions, it's way too broad.

It's very unlikely that this one would throw an exception though, so I think it's safe. Generally, nullary functions with no side effects should always be properties, and I'd like to stick to that. I suppose a case could be made that bona fide exceptions (not assertions) could be counted as side effects.

Reply:
What is the advantage of having it be a property and not a function?